bundle/tests/scorecard/config.yaml

```diff
@@ -12,42 +12,66 @@ stages:
     labels:
       suite: basic
```

Comment:
These first three files seem to have been changed due to a newer operator-sdk version

Reply:
We should go through and ensure we are all running the same version - it's been a while since we did an upgrade - there are generally changes we need to make to the code base after an upgrade - we should get a card in place to do this

Reply:
operator-sdk is pretty easy to get the old version - but we _should_ look at updating in clowder - can you get a card together for an operator-sdk update in clowder?